pkgs/development/tools/ansible-lint/default.nix

```diff
@@ -11,9 +11,9 @@ pythonPackages.buildPythonPackage rec {
     sha256 = "0cnfgxh5m7alzm811hc95jigbca5vc1pf8fjazmsakmhdjyfbpb7";
   };
 
-  propagatedBuildInputs = with pythonPackages; [ pyyaml six ] ++ [ ansible ];
+  propagatedBuildInputs = with python3Packages; [ pyyaml six ] ++ [ ansible ];
```

Comment:
... and `ansible` by `ansible.override { python3 = python3; }`.
That way, both are using the same set of python packages.